lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -93,10 +122,16 @@ export default function NxTransferList<T extends string | number>(props: Props<T
                            isSelected={true}
                            items={selected}
                            onItemChange={onChange}
+                           onReorderItem={onReorderItem}
                            footerContent={selectedItemsCountFormatter(selectedCount)}
-                           filterFn={filterFn} />
+                           filterFn={filterFn}
+                           allowReordering={allowReordering} />
     </div>
   );
 }
 
 NxTransferList.propTypes = propTypes;
+
+NxTransferList.defaultValues = {
```

Comment:
When I try to google this ('react "defaultValues"') all I get are results for some third-party form hook.  Are you sure this is a thing? Even if so, it's not something that we use anywhere else.

Reply:
> When I try to google this ('react "defaultValues"') all I get are results for some third-party form hook. Are you sure this is a thing? Even if so, it's not something that we use anywhere else.

Removed it here: 774e9702b6a1123bac0705354974edd6087775a5